rafthttp/stream.go

```diff
@@ -51,6 +51,7 @@ var (
 		"2.3.0": {streamTypeMsgAppV2, streamTypeMessage},
 		"3.0.0": {streamTypeMsgAppV2, streamTypeMessage},
 		"3.1.0": {streamTypeMsgAppV2, streamTypeMessage},
+		"3.2.0": {streamTypeMsgAppV2, streamTypeMessage},
```

Comment:
i do not think we need streamTypeMessage anymore. can you get rid of it and test it? if it can still work with 3.1 + 3.0, then it is fine.